### DISCUSSION TITLE
Where are exes stored ?

### DISCUSSION THREAD

**Discussion by Methen** [LegacyUpdate/LegacyUpdate#178] [Questions & Help] [answered]
Hello, I am trying legacy on my xp pro I was wondering where are all the exes stored when they are down loaded I like to save them ?

**Comment by kirb** [comment, marked as answer]
The Legacy Update setup stores files in C:\\Documents and Settings\\All Users\\Application Data\\Legacy Update (C:\\ProgramData\\Legacy Update on Vista+) but note this folder is deleted when setup finishes.

Windows Update stores files in C:\\Windows\\SoftwareDistribution\\Download. The files here won’t have extensions, which is annoying. In the case of 2000/XP you can probably assume most are .exe or .psf (patch data files). 

Note that files WU download are often “express”, and not the same as the “full” download you see on the Windows Update Catalog or the support website/download center. They may not work if the system isn’t in an expected prerequisite state to qualify for the express install, and may require internet to download further files.

**Reply by sal-ammoniac** [reply]
yeah and psf requires extra tool to decompress

**Reply by Methen** [reply]
You miss understood I mean all the windows update files where are they stored on xp ?

**Reply by kirb** [reply]
When you download updates, the downloaded files get stored in C:\\Windows\\SoftwareDistribution\\Download, until they get cleaned up at some later point. I think that answers your question, sorry if I’m still not understanding what you’re looking for.
